examples/react/getting-started/src/App.tsx

```diff
@@ -58,7 +60,11 @@ export function App() {
                 <Pagination />
               </div>
               <div>
-                <TrendingItems itemComponent={ItemComponent} limit={4} />
+                <TrendingItems
+                  itemComponent={ItemComponent}
+                  limit={6}
+                  layoutComponent={(props) => <Carousel {...props} />}
```

Comment:
as props is just forwarded, I think this can be
```suggestion
                  layoutComponent={Carousel}
```

Reply:
Done!